estate/models/estate_property.py

```diff
@@ -0,0 +1,24 @@
+from odoo import models, fields
+from dateutil.relativedelta import relativedelta
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = "Properties in sale."
+
+    name = fields.Char(required=True)
+    description = fields.Char(help=None)
+    postcode = fields.Char(help=None)
+    date_availability = fields.Date(default=lambda self: fields.Datetime.now() + relativedelta(months=3), copy=False)
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer(help=None)
+    facades = fields.Integer(help=None)
+    garage = fields.Boolean(help=None)
+    garden = fields.Boolean(help=None)
+    garden_area = fields.Integer(help=None)    
+    garden_orientation = fields.Selection(
+        selection=[('North', 'North'), ('South', 'South'), ('East', 'East'), ('West','West')])
```

Comment:
basic thing to differentiate the key to its value, is the capital letter
('north', 'North') => here it's not really important but it could be useful in a case like
('ongoing', 'The offer is currently in ongoing phase') ==> we refer to it by it's key